text/0000-slack.md

```diff
@@ -0,0 +1,105 @@
+
+As a part of CNCF slack onboarding, we can also migrate all of our data and channels onto the CNCF workspace, preserving any past history of conversations that took place in the Buildpacks slack. For more details please see [CNCF - Migrating workspaces](https://slack.com/intl/en-gb/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another)
+
+# What it is
+[what-it-is]: #what-it-is
+
+This RFC proposes that we move the Buildpacks slack instance from an independent instance to the CNCF instance.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+The maintainers will have to request the migration of the Buildpacks slack over to CNCF slack and co-ordinate the announcements/user migration.
+
+The following are the slack channels that will be migrated over and their proposed names after migrations - 
+
+| Current                 | Proposed                       |
+| ----------------------- | ------------------------------ |
+| announcements           | buildpacks-announcements       |
+| arm                     | buildpacks-arch-arm            |
+| buildpacks-authors      | buildpacks-authors-team        |
+| buildpacks-distribution | buildpacks-distribution        |
+| general                 | buildpacks                     |
+| implementation          | buildpacks-implementation-team |
+| implementation-ops      | buildpacks-implementation-ops  |
+| learning                | buildpacks-learning-team       |
+| learning-ops            | buildpacks-learning-ops        |
+| maintainers             | buildpacks-maintainers         |
+| mentoring               | buildpacks-mentoring           |
+| ops                     | buildpacks-ci-ops              |
+| pack-cli                | buildpacks-pack-cli            |
+| platform                | buildpacks-platform-team       |
+| registry                | buildpacks-registry            |
+| spec                    | buildpacks-spec                |
+| spec-ops                | buildpacks-spec-ops            |
+| tekton                  | buildpacks-platform-tekton     |
+| windows                 | buildpacks-arch-windows        |
+
+The following channels will not be migrated - 
+
+- classic (inactive)
+- community (inactive)
+- delivery (inactive)
+- docs (inactive)
+- elixir (inactive)
+- governance (inactive)
+- java (inactive)
+- kaniko (inactive)
+- kpack (not part of the official project)
+- packfile (inactive)
+- python (inactive)
+- random (inactive)
+- rfcs (inactive)
+- ruby (inactive)
+- stacks (inactive)
+- summit-2021 (inactive)
+- user-research (inactive)
+- wg-chat (inactive)
+
```

Comment:
```suggestion
## Migration Steps
* [ ] once RFC finalizes, announce migration in slack and mailing list with move over date.
* [ ] before Feb 2, get slack backup of history
* [ ] on date move over
  * [ ] setup final announcement in slack and set all channels as read-only.
  * [ ] backup history merge with previous backup, give to CNCF for migration
```

Reply:
@samj1912 can you ask how long the migration will take to cutover?

Reply:
Hearing back, it should only take a few minutes. We'll plan for an hour maintenance.